docs/dev/run_the_etl.rst

```diff
@@ -0,0 +1,88 @@
+.. _run-the-etl:
+
+=======================
+Walk, Don't Run the ETL
+=======================
+
+So you want to run the PUDL data processing pipeline? This is the most involved
+way to get access to PUDL data. It is only suggested if you want to edit the
+ETL process or contribute to the code base. Check the basic usage page for the
+other access methods (see: :ref:`basic-usage`). Measure twice, cut once.
+
+These instructions assume you have already gone through the development setup
+(see: :ref:`dev_setup`).
+
+There are four main scripts that are involved in the PUDL processing pipeline:
+
+1. `ferc1_to_sqlite` :doc: clones FERC Form 1's dbf files <clone_ferc1> as
+   a sqlite database - a necessary prep extract step for FERC Form 1.
+2. `pudl_etl` is where the magic happens. This is the main script which
+   coordinates the "Extract, Transform, Load" steps which results in
+   `Tabular Data Packages <https://frictionlessdata.io/specs/tabular-data-package/>`_.
+3. `datapkg_to_sqlite` converts the Tabular Data Packages into a SQLite
+   database. We recommend this for all of the small-medium tables which is
+   currently everything but the hourly EPA CEMS data.
+4. `epacems_to_parquet` converts the Tabular Data Packages into parquet files
+   for the large datasets - which is currently the hourly EPA CEMS data.
+
+Settings files dictate which datasets, years, tables or states get processed in
+this processing pipeline. Two examples are provided in the ``settings`` folder
+that is created when you run ``pudl_setup`` (see: :ref:`install-workspace` for
+setup tools and see: :ref:`settings_files` for info on settings files). If you
+want to run an ETL with different configurations, feel free to edit or
+duplicate these files,
```

Comment:
Probably want to note here that users should ONLY expect these two configurations to work (most recent year, or all years) since we've gotten into a lot of support problems with people trying to run other combinations of things.